example/rpc_client.t/run.t

```diff
@@ -0,0 +1,15 @@
+Start dune in watch mode, sending errors to /dev/null to surpress the alerts
+about using the unstable module Dune_rpc.
+  $ dune build -w 2> /dev/null &
+
+Wait for the program produced by the above step to exist.
+  $ while ! test -f _build/default/rpc_client.exe; do sleep 1; done
+
+Run the program. The program takes care of shutting down the build server.
+  $ _build/default/rpc_client.exe
+  Sending ping to server...
+  Got response from server...
+  Creating progress stream...
+  Waiting for next progress event...
+  Got progress_event: Success
+  Shutting down RPC server...
```

Comment:
Should these lines of code be wrapped in a code block?

Reply:
What do you mean by code block? Just clarifying that this file defines a test; it's not just documentation.

Reply:
I mean in monospace: 
```
$ _build/default/rpc_client.exe
  Sending ping to server...
  Got response from server...
  Creating progress stream...
  Waiting for next progress event...
  Got progress_event: Success
  Shutting down RPC server...
  ```

Reply:
Are you assuming that this is a markdown file? This file is a cram test. It's a script that runs a test with inline comments, commands, and their expected output. It doesn't get rendered by github.

Reply:
OH! Thank you for that clarification! My mistake and oversight.